test/ueberauth_oidc/strategy/oidc_test.exs

```diff
@@ -17,17 +17,17 @@ defmodule Ueberauth.Strategy.OIDCTest do
     setup_with_mocks [
       {OpenIDConnect, [],
        [
-        authorization_uri: fn
-          "test_provider", %{redirect_uri: redirect_uri} = params ->
-            state_param =
-              case params[:state] do
-                nil -> ""
-                state -> "&state=#{state}"
-              end
-
-            "#{@request_uri}?redirect_uri=#{redirect_uri}#{state_param}"
-        end,
-        fetch_tokens: fn _, _ -> @valid_tokens end,
+         authorization_uri: fn
```

Comment:
It looks like the changes in lines 20–62 here are formatting changes automatically applied by my editor. Let me know if you'd prefer me to pull these out, I'd be happy to do so.

Reply:
This is fine, it lines out correctly now.